.travis.yml

```diff
@@ -43,11 +43,16 @@ jobs:
     - <<: *style_python
       name: PY@3.6 +PEP8
       python: "3.6"
+    - <<: *style_python
+      name: PY@3.7 +PEP8
+      python: "3.7"
+      dist: xenial
+      sudo: true
     # - &style_cpp
     #   stage: 'Style'
     #   name: clang-format@5.0.0
     #   language: python
-    #   python: "2.7"
```

Comment:
Python 3 is only needed if the python bindings are build

Reply:
Well in this case, we require Python due to `run-clang-format.py`. It's 2019 already and I'd say it's time to leave 2.7 behind altogether.

Reply:
Ok, then  you only need to change the entries in the matrix that use `run-clang-format.py` if it is not backwards compatible :)
But I think all of this can be done in another PR and is independent of this change, no?